cdc/kv/region_worker.go

```diff
@@ -273,7 +273,7 @@ func (w *regionWorker) handleSingleRegionError(ctx context.Context, err error, s
 	}
 
 	revokeToken := !state.initialized
-	err2 := w.session.onRegionFail(ctx, regionErrorInfo{
+	err2 := w.session.onRegionFail(w.parentCtx, regionErrorInfo{
```

Comment:
Do we need to add some comments here? I see another place where it is used added a comment why we use parentCtx instead of ctx.

See: https://github.com/pingcap/ticdc/blob/37bac66f0673103892c71e585ab3d9d4658c1f74/cdc/kv/region_worker.go#L792 
It looks like the changes here are also for this purpose, sorry I'm not very familiar with this piece of code.

Reply:
yes, the same reason